src/wizard/wizard-dialog.ts

```diff
@@ -327,93 +328,74 @@ export class ESPHomeWizardDialog extends LitElement {
     return [heading, content, hideActions];
   }
 
-  private _renderPickBoard() {
+  private _renderPickPlatform() {
+    const platforms: { [key in SupportedPlatforms]: string } = {
```

Comment:
Let's merge this with the default boards object.

Reply:
Moved platform titles & default boards to `const.ts`.